modules/tictactoe/tictactoe.js

```diff
@@ -255,24 +256,24 @@ exports.run = function(api, event) {
 
         output = checkWin(game, event.sender_name.trim());
         if (output) {
-            api.sendMessage("Game Over! " + output + " wins!", event.thread_id);
+            api.sendMessage('Game Over! ' + output + ' wins!', event.thread_id);
             removeGame(game);
         } else {
-            api.sendMessage(game.turn + "'s turn!", event.thread_id);
+            api.sendMessage(game.turn + '\'s turn!', event.thread_id);
         }
     }
 
-    //If we are surrendering
-    if (command.startsWith(commandPrefix + "surrender")) {
+    // If we are surrendering
+    if (command.startsWith(commandPrefix + 'surrender')) {
         var game = findGame(event.thread_id, event.sender_name.trim());
 
         if (!game) {
-            api.sendMessage("You're not even in a game! Why are you surrendering?");
+            api.sendMessage('You\'re not even in a game! Why are you surrendering?');
             return;
         }
 
         removeGame(game);
-        
-        api.sendMessage(event.sender_name.trim() + " surrenders! " + (event.sender_name.trim() !== game.noughts ? game.noughts : game.crosses) + " wins!");
+
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/theprojectshop/Kassy/file/2529976290/issues/source?bid=3282987&fileBranchId=3282998#l277)